qa/suites/rados/upgrade/jewel-x-singleton/8-workload/rgw-swift.yaml

```diff
@@ -3,8 +3,7 @@ meta:
    swift api tests for rgw
 tasks:
 - rgw: 
-    client.0:
-    default_idle_timeout: 300
+    client.0: null
```

Comment:
it's safe to put 
```yaml
- rgw:
    client.0:
```

the empty value will be interpreted as an empty value, i.e. `null`, see http://www.yaml.org/spec/1.2/spec.html#e-scalar. and https://github.com/ceph/ceph/blob/master/qa/tasks/rgw.py#L186 also documents this usage.